packages/ui/src/ui/pages/query-tracker/QueryResultsVisualization/preparers/prepareWidgetData.ts

```diff
@@ -0,0 +1,26 @@
+import _ from 'lodash';
+import type {ChartKitWidgetData} from '@gravity-ui/chartkit';
+import {prepareBar} from './bar';
+import {prepareScatter} from './scatter';
+import type {PrepareLineArgs} from './types';
+import {buildD3Config} from './d3config';
+
+export const prepareWidgetData = (args: PrepareLineArgs): ChartKitWidgetData => {
+    const d3Config = buildD3Config(args);
+
+    let data;
```

Comment:
const data = args.visualization.id === 'scatter' ? prepareScatter(args) : prepareBar(args);

Reply:
We might to have more visualisations here in the future, lets leave it as is for future extending?